corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -148,47 +148,59 @@ hqDefine('cloudcare/js/utils', [
         return new RegionContainer();
     };
 
-    var showLoading = function () {
+    var showProminentLoading = function () {
         hqRequire([
             "cloudcare/js/formplayer/app",
-            "hqwebapp/js/toggles",
             "cloudcare/js/formplayer/layout/views/progress_bar",
-        ], function (FormplayerFrontend, toggles, ProgressBar) {
-            if (toggles.toggleEnabled('USE_PROMINENT_PROGRESS_BAR')) {
-                const progressView = ProgressBar({
-                    progressMessage: gettext("Loading..."),
-                });
-                if (!FormplayerFrontend.regions) {
-                    FormplayerFrontend.regions = getRegionContainer();
-                }
-                $('#breadcrumb-region').css('z-index', '0');
-                const loadingElement = FormplayerFrontend.regions.getRegion('loadingProgress');
-                loadingElement.show(progressView);
-                let currentProgress = 10;
-                progressView.progressEl.find('.progress').css("height", "12px");
-                progressView.progressEl.find('.progress-container').css("width", "50%");
-                progressView.progressEl.find('.progress-title h1').css("font-size", "25px");
-                progressView.progressEl.find('#formplayer-progress ').css("background-color", "rgba(255, 255, 255, 0.7)");
-                progressView.setProgress(currentProgress, 100, 200);
-                sessionStorage.progressIncrementInterval = setInterval(function () {
-                    if (currentProgress <= 100) {
-                        progressView.setProgress(currentProgress, 100, 200);
-                        currentProgress += 1;
+        ], function (FormplayerFrontend, ProgressBar) {
+            setTimeout(function () {
+                const formplayerQueryInProgress = sessionStorage.formplayerQueryInProgress && JSON.parse(sessionStorage.formplayerQueryInProgress);
+                if (formplayerQueryInProgress) {
+                    const progressView = ProgressBar({
+                        progressMessage: gettext("Loading..."),
+                    });
+                    if (!FormplayerFrontend.regions) {
+                        FormplayerFrontend.regions = getRegionContainer();
                     }
-                }, 250);
+                    $('#breadcrumb-region').css('z-index', '0');
+                    const loadingElement = FormplayerFrontend.regions.getRegion('loadingProgress');
+                    loadingElement.show(progressView);
+                    let currentProgress = 10;
+                    progressView.progressEl.find('.progress').css("height", "12px");
```

Comment:
nit: can this logic be moved to `progress_bar.js`

Reply:
that was something we talked about in the previous PR https://github.com/dimagi/commcare-hq/pull/34158#discussion_r1513317687 Long story short, those style changes were getting overwritten.

Reply:
I see, that's when it was put in `initialize`? What do you think about method that's like `setProminentLoadingStyle`?

Reply:
yeah, I can add a function to handle updating these styles

Reply:
added here https://github.com/dimagi/commcare-hq/pull/34286/commits/455464223047dd05e6e5056ba37359ddabb583ed

Reply:
Sorry, I actually meant adding that method directly in `progress_bar.js`. Otherwise, I think how it was originally is good as is.

Reply:
ah, ok. reverting to how it was.